scripts/ripe-atlas

```diff
@@ -1,94 +1,10 @@
 #!/usr/bin/env python
-
-import importlib
-import os
-import pkgutil
-import re
 import sys
 
-from ripe.atlas.tools import commands
+from ripe.atlas.tools.script import RipeAtlas
```

Comment:
I don't understand why you'd move so 95% of this file into a new file when it doesn't change the modularity.  Is this in anticipation for something else happening here?